### PR TITLE
Reco material tools

### PR DESCRIPTION
**Pull request by rovere**
- Added new tools to easily compare the Material map between MC and reconstruction.
- Locally store an improved version of the reconstruction material description, but do not propagate it to "production". It will very likely be bundled with the next round of material description for PhaseI after the PhaseI geometry will be finalised.
- Extensive documentation on how to run and use the tool has been included.

### CODE REVIEW

**Comment by civanch** [comment]
please test

File: SimTracker/TrackerMaterialAnalysis/plugins/TrackingMaterialAnalyser.cc [hunk 1]
```diff
@@ -178,6 +178,15 @@ void TrackingMaterialAnalyser::split( MaterialAccountingTrack & track )
   for (unsigned int i = 0; i < track.detectors().size(); ++i)
     group[i] = findLayer( track.detectors()[i] );
 
+  for (unsigned int i = 0; i < group.size(); ++i)
+    if (group[i] > 0)
+      std::cout << "For detector i: " << i << " index: "
```
Comment (dmitrijus) [inline SimTracker/TrackerMaterialAnalysis/plugins/TrackingMaterialAnalyser.cc]: Can this perhaps become a LogInfo?

**Comment by ianna** [comment]
@rovere - does it mean we need to regenerate and update DB payloads?

**Comment by ianna** [comment]
+1

**Comment by rovere** [comment]
@ianna - not at the moment. This PR contains a better material description for the SiStrip Tracker, but I did not couple that with the last one you pushed related to the Pixel for PhaseI.
My idea at that time was that in any case we are still waiting for the final PhaseI simulation geometry ==> we anyway have to re-derive the material description for the reconstruction one and, at than point, being the final simulation geometry for 2017 data taking, I can couple all the changes in one go.
So, the bottom line is that I do expect to have an update on the material description as soon as the PhaseI simulation geometry is delivered: at that point I think we need to regenerate and update the DB payloads, modulo a successful validation.

**Comment by rovere** [comment]
HI all,
this PR should indeed go into the release (81X) as soon as possible, so that we have at hand all the tools we need when the PhaseI pixel geometry will come.

Thanks.

**Comment by dmitrijus** [comment]
+1

**Comment by ianna** [comment]
+1

**Comment by civanch** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**